src/monodroid/jni/dylib-mono.h

```diff
@@ -344,7 +344,9 @@ enum MonoAotMode {
 	MONO_AOT_MODE_HYBRID,
 	/* Enables full AOT mode, JIT is disabled and not allowed,
 	 * equivalent to mono_jit_set_aot_only (true) */
-	MONO_AOT_MODE_FULL
+	MONO_AOT_MODE_FULL,
+
+	MONO_AOT_MODE_UNKNOWN = 0xBADBAD
```

Comment:
Will this value be passed into mono, and if so, what will happen?

Reply:
It won't be passed to Mono (https://github.com/grendello/xamarin-android/blob/runtime-profile/src/monodroid/jni/monodroid-glue.cc#L1904-L1905)